site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -80,7 +80,15 @@ gcloud compute firewall-rules create game-server-firewall \
 
 ## Creating the cluster
 
-A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster. 
+
+{{% feature publishVersion="1.29.0" %}}
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+For more information about choosing the best mode for your needs, refer to
+[Choose a GKE mode of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/choose-cluster-mode).
```

Comment:
I feel like for game server workloads several of the items on this doc aren't as applicable.

Can we include the content we have on our internal guide (not just here, but throughout the document)? I don't think we have to remove the link we have here, but it has some really good workload specific context, limitation and differences in workload requirements -- none of which is covered here.

Reply:
Hmmm. We could do a section and context on this page that covers the TL:DR of choosing a mode specifically for Agones (taken from our internal guide) and keep this link for more general info about choosing a mode.

If at some point GKE uses that internal guide to create a public doc focused on gaming we can trim the fat from this page and link there instead.

Reply:
> Hmmm. We could do a section and context on this page that covers the TL:DR of choosing a mode specifically for Agones (taken from our internal guide) and keep this link for more general info about choosing a mode.

SGTM.

> If at some point GKE uses that internal guide to create a public doc focused on gaming we can trim the fat from this page and link there instead.

Also SGTM 👍🏻

Reply:
Okay, I added this section in lines 69-107, could you and @zmerlynn take a peek?